website/docs/r/one_dashboard.html.markdown

```diff
@@ -357,7 +368,62 @@ Each widget type supports an additional set of arguments:
     * `initial_sorting` - (Optional) An attribute that describes the sorting mechanism for the table. This attribute requires specifying the following attributes in a nested block -
         * `name` - (Required) The name of column to be sorted. Examples of few valid values are `timestamp`, `appId`, `appName`, etc.
         * `direction` - (Required) Defines the sort order. Accepted values are `asc` for ascending or `desc` for descending.
+    * `data_format` - (Optional) A nested block that describes data format. See [Nested data_format blocks](#nested-data_format-blocks) below for details.
+        
+
+### Nested `data_format` blocks
+
+Nested `data_format` specifies the data structure format in the table. It defines how the data is organized and interpreted.This attribute requires specifying the following attributes in a nested block -
+
+  * `name` - (Required) This attribute mandates the specification of the column name to be formatted. It identifies which column the data format should be applied to.
+  * `type` - (Required) This attribute sets the format category for your data. Accepted values include `decimal` for numeric values, `date` for date/time values, `duration` for length of time, `recent-relative` for values referencing a relative point in time, `custom` for specific date/time formats and `humanized` to set autoformat as true.
+  * `format` - (Optional) This attribute is provided when the `type` attribute is set to `date`, defining the specific date format to be applied to your data.
+
+  |     Accepted value  |        Format           |                     
+  |---------------------|-------------------------| 
+  | `%b %d, %Y`         | `MMM DD,YYYY`           | 
+  | `%d/%m/%Y`          | `DD/MM/YYYY(EU)`        | 
+  | `%x`                | `DD/MM/YYYY(USA)`       | 
+  | `%I:%M%p`           | `12-hour format`        | 
+  | `%H:%Mh`            | `24-hour format`        | 
+  | `%H:%Mh UTC (%Z)`   | `24-hour with timezone` | 
+  | `%Y-%m-%dT%X.%L%Z`  | `ISO with timezone`     | 
+  | `%b %d, %Y, %X`     | `MMM DD, YYYY,hh:mm:ss` | 
+  | `%X`                | `hh:mm:ss`              | 
+
+-> **IMPORTANT!**
+    `type` should be set to `custom` inorder to use the formats listed above.
```

Comment:
what we probably can do for greater clarity is the following - 
- add sample code snippets (not of the entire widget, just the `data_format{}` portion for all cases of the three types we currently have, covering edge cases i.e. the duration type, the date/time type with no format and with a format, and the number type with decimals, with autoformat, etc.)
- add something like "see examples below" in the description of all of these attributes we've listed above; or even better, refactor this `IMPORTANT!` message to state these edge cases and suggest the customer to see code snippets linked below

Reply:
@pranav-new-relic add the code snippets as discussed and suggested .